### PR TITLE
Refactor/move storybook to nx

### PR DESCRIPTION
**Pull request by gbonumore**

### CODE REVIEW

File: tsconfig.base.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@monorepo/shared/ui-library": ["libs/shared/ui-library/src/index.ts"]
```
Comment (adam-arold) [inline tsconfig.base.json:19]: What is suggested by Nx as a best practice is to use the directory + library name as a path like `@shared/ui-library`.

**Review by adam-arold — COMMENTED** [review]
LGTM! I have a tiny comment in there, but I think this is 👌

File: tsconfig.base.json (hunk 1 above)
Comment (gbonumore) [reply tsconfig.base.json:19]: this was generated and haven't thought too much about that will update `@monorepo/shared/ui-library` into `/shared/ui-library` then!

**Review by jordaniza — APPROVED** [review]
Transferring the old ui-components library into shared/ui-library makes sense to me. The changes you have put forward all seem as expected. Approved.

**Review by jordaniza — APPROVED** [review]
Transferring the old ui-components library into shared/ui-library makes sense to me. The changes you have put forward all seem as expected. Approved.

**Review by adam-arold — APPROVED** [review]
LGTM!

**Pull request merged**